### PR TITLE
Add path /var/run/secrets/.../serviceaccount/token

### PR DESCRIPTION
**Pull request by guettli**
Add path /var/run/secrets/kubernetes.io/serviceaccount/token so that new users understand the "magic".

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ You can access the API from inside a pod using automatically mounted service acc
 The API permissions of the service account depend on the
 [authorization plugin and policy](/docs/reference/access-authn-authz/authorization/#authorization-modules) in use.
 
-In version 1.6+, you can opt out of automounting API credentials for a service account by setting `automountServiceAccountToken: false` on the service account:
+In version 1.6+, you can opt out of automounting API credentials on `/var/run/secrets/kubernetes.io/serviceaccount/token` for a service account by setting `automountServiceAccountToken: false` on the service account:
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:45]: Also OK
```suggestion
You can opt out of automounting API credentials on `/var/run/secrets/kubernetes.io/serviceaccount/token` for a service account by setting `automountServiceAccountToken: false` on the ServiceAccount:
```

**Review by sftim — COMMENTED** [review]
@guettli would you be willing to revise this page more thoroughly? Every mention of “version 1.6+” is now stale because version 1.5 and older of Kubernetes are very much not supported any more.

**Comment by zacharysarah** [comment]
> @guettli would you be willing to revise this page more thoroughly? Every mention of “version 1.6+” is now stale because version 1.5 and older of Kubernetes are very much not supported any more.

That would definitely be a valuable update. That said, I'd rather approve a single incremental improvement and invite others to follow.

/lgtm
/approve

**Pull request merged**